### PR TITLE
Allow sandbox to work with collection variables by exposing `collectionVariables` in execution

### PR DESCRIPTION
**Pull request by kamalaknn**

### CODE REVIEW

File: test/unit/sandbox-libraries/pm.test.js [hunk 1]
```diff
@@ -19,6 +19,14 @@ describe('sandbox library - pm api', function () {
                 value: 2.5,
                 type: 'number'
             }],
+            collectionVariables: [{
+                key: 'var1',
+                value: 'collection-var1',
+                type: 'string'
+            }, {
+                key: 'var3',
+                value: 'collection-var3'
```
Comment (kunagpal) [inline test/unit/sandbox-libraries/pm.test.js]: Any specific reason for skipping the `type` here?

File: test/unit/sandbox-libraries/pm.test.js [hunk 2]
```diff
@@ -497,6 +505,82 @@ describe('sandbox library - pm api', function () {
...
+
+            context.execute(`
+                var assert = require('assert');
+
+                assert.strictEqual(pm.variables.get('var1'), 1);
+                assert.strictEqual(pm.variables.get('var2'), 'two');
+
+                // can set variables with same keys as collectionVariables
+                assert.strictEqual(pm.variables.get('var3', undefined));
+                assert.strictEqual(pm.variables.set('var3', 'hola'));
```
Comment (kunagpal) [inline test/unit/sandbox-libraries/pm.test.js]: ?

File: test/unit/sandbox-libraries/pm.test.js (hunk 1 above)
Comment (kamalaknn) [reply test/unit/sandbox-libraries/pm.test.js]: yup, I threw in some of these randomly to make sure legacy variables work

File: test/unit/sandbox-libraries/pm.test.js (hunk 2 above)
Comment (kamalaknn) [reply test/unit/sandbox-libraries/pm.test.js]: I might have accidentally removed `var3` from context, adding it back now. Good spot.

**Comment by kamalaknn** [comment]
@kunagpal addressed your comments 🙂

**Pull request merged**